src/features/navigator/RefreshButton.tsx

```diff
@@ -1,20 +1,30 @@
-import { FC } from 'react';
+import { FC, MouseEvent } from 'react';
 import { faRefresh } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon as FAIcon } from '@fortawesome/react-fontawesome';
 import { Button } from '../../common/components';
+import type { AppDispatch } from '../../app/store';
 import { useAppDispatch } from '../../common/hooks';
-import { setFresh } from './navigatorSlice';
+import { clearCacheAction } from '../../common/api/searchApi';
 import classes from './Navigator.module.scss';
 
-const RefreshButton: FC = () => {
+// This handler factory pattern is primarily to aid unit testing.
```

Comment:
[Use](https://stackoverflow.com/questions/39755439/how-to-mock-imported-named-function-in-jest-when-module-is-unmocked/53402206#53402206) `jest.requireActual(moduleName)` instead to make this code simpler.

Reply:
Actually, `<RefreshButton />` does not need to accept a separate handler at this time in any case.